presto-hive/src/main/java/io/prestosql/plugin/hive/s3/PrestoS3FileSystem.java

```diff
@@ -1255,6 +1264,16 @@ private void uploadObject()
                             break;
                     }
                 }
+                switch (s3StorageClass) {
+                    case Standard:
+                        request.withStorageClass(Standard);
```

Comment:
As you now have `com.amazonaws.services.s3.model.StorageClass` object at hand (the `s3StorageClass`), this can now be:

```
request.withStorageClass(s3StorageClass);
```

-- without a `switch`